en/advanced-gimbal-control.md

```diff
@@ -1,20 +1,24 @@
 # Gimbal Control Setup
 
-PX4 contains a generic mount/gimbal control driver with different input and output methods. Any input can be selected to drive any output.
-
-First, make sure the driver runs, using `vmount start`, then configure its parameters.
+PX4 contains a generic mount/gimbal control driver with different input and
+output methods. Any input method can be selected to drive any output.
 
 ## Parameters
-The parameters are described in [src/drivers/vmount/vmount_params.c](https://github.com/PX4/Firmware/blob/master/src/drivers/vmount/vmount_params.c). The most important ones are the input (`MNT_MODE_IN`) and the output (`MNT_MODE_OUT`) mode. By default, the input is disabled. Any input method can be selected to drive any of the available outputs.
-
-If a mavlink input mode is selected, manual RC input can be enabled in
-addition (`MNT_MAN_CONTROL`). It is active as long as no mavlink message is received yet, or mavlink explicitly requests RC mode.
-
+The parameters are described [here](parameter_reference.md#mount). The most
+important ones are the input (`MNT_MODE_IN`) and the output (`MNT_MODE_OUT`)
+mode. By default, the input is disabled. After selecting the input mode, reboot
+the vehicle so that the mount driver starts. Any input method can be selected to
+drive any of the available outputs.
 
+If the input mode is set to `AUTO`, the mode will automatically be
+switched based on the latest input. To switch from mavlink to RC, a large stick
+motion is required.
 
-### Configure the gimbal mixer for AUX output
```

Comment:
So what does it mean to configure it for AUX output? Is there any other way you can configure the mixer? Why woudl you do this.